recipes/helics/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "helics" %}
+{% set version = "1.2.1" %}
+{% set checksum = "0786edb14f854d3c54da9882dcaaaec56d2370ec98abef44d211655065194e5f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/GMLC-TDC/HELICS-src/archive/v{{ version }}.tar.gz
+  sha256: {{ checksum }}
+
+build:
+  number: 0
+  skip: True  # [linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - swig
+    - ninja
+  host:
+    - python
+    - pip
```

Comment:
let's wait the build in our CI finish to test one thing. You may have to change the ``host`` dependencies to ``build``. Because of your ``CMakeList`` the ``python`` and ``pip`` have to be in ``build`` and not in ``host``